src/utils/ethereumUtils.ts

```diff
@@ -51,46 +49,32 @@ import { Navigation } from '@/navigation';
 import { parseAssetNative } from '@/parsers';
 import store from '@/redux/store';
 import {
-  ARBITRUM_BLOCK_EXPLORER_URL,
   ARBITRUM_ETH_ADDRESS,
-  chains,
   ETH_ADDRESS,
   ethUnits,
   MATIC_MAINNET_ADDRESS,
   MATIC_POLYGON_ADDRESS,
   BNB_BSC_ADDRESS,
-  OPTIMISM_BLOCK_EXPLORER_URL,
   OPTIMISM_ETH_ADDRESS,
   optimismGasOracleAbi,
   OVM_GAS_PRICE_ORACLE,
-  POLYGON_BLOCK_EXPLORER_URL,
-  BSC_BLOCK_EXPLORER_URL,
   BNB_MAINNET_ADDRESS,
 } from '@/references';
 import Routes from '@/navigation/routesNames';
 import { logger, RainbowError } from '@/logger';
 import { IS_IOS } from '@/env';
+import { RainbowNetworks, getNetworkObj } from '@/networks';
 
+// hate this pattern but could be messy
```

Comment:
Lol any more clarification needed?

Reply:
will update linking to this ticket, added some context here: https://linear.app/rainbow/issue/APP-631/remove-networks-from-assettype